UIU-3373: Proxy Borrower field does not equal Unknown user (#2919)
folio-org/ui-users: CHANGELOG.md

* Update patron block message to support pluralization. Refs UIU-3394.
* Change fields order in `ExtendedInfo` and `RequestPreferencesView` components. Refs UIU-3122.
* Extended information - change order of fields, use MultiSelection for the departments field. Refs UIU-3123.

> * Proxy Borrower field does not equal Unknown user. Refs UIU-3373.

Reviewer: ```suggestion
* Display "Unknown user" instead of dash in proxy borrower field. Refs UIU-3373.
```
